include/mxnet/c_api.h

```diff
@@ -585,6 +585,8 @@ MXNET_DLL int MXNDArrayCreate(const uint32_t *shape,
 
 /*!
  * \brief create a NDArray with specified shape and data type
+ *  This api is available when MXNet is built with flag
+ *  USE_INT64_TENSOR_SIZE=0 (by default)
```

Comment:
I think instead of adding the comment, we should just error out from the API when the incorrect flags are used with API.

Reply:
There already is a functionality to throw error messages when usage is of large tensor and mxnet is not built with USE_INT64_TENSOR_SIZE=1. This is a documentation for devs to help them decide which one to choose.